nih-sparc/sparc-app
# Wire Up Simulation Page #4a9j8g #4a9j8q

# Description

The purpose of this PR is to wire up the simulation details page. 

## Type of change


# How Has This Been Tested?

1. Navigate to the Find Data > Simulations tab.
2. Click on any simulation dataset. All the details should be displayed and populated as expected. 
3. Navigate to Find Data > Datasets tab. Click on any regular dataset and verify that it's populated with the right dataset data.

# Checklist:
